EIPS/eip-3529.md

```diff
@@ -28,13 +28,16 @@ Gas refunds for SSTORE and SELFDESTRUCT were originally introduced to motivate a
 | Constant | Value |
 | - | - |
 | `FORK_BLOCK` | TBD |
-| `NEW_MAX_REFUND_QUOTIENT` | 5 |
+| `MAX_REFUND_QUOTIENT` | 5 |
 
 For blocks where `block.number >= FORK_BLOCK`, the following changes apply.
 
 1. Remove the `SELFDESTRUCT` refund.
 2. Replace `SSTORE_CLEARS_SCHEDULE` (as defined in [EIP-2200](./eip-2200.md)) with `SSTORE_RESET_GAS + ACCESS_LIST_STORAGE_KEY_COST` (4,800 gas as of [EIP-2929](./eip-2929.md) + [EIP-2930](./eip-2930.md))
-3. Reduce the max gas refunded after a transaction to `gas_used // NEW_MAX_REFUND_QUOTIENT`
+3. Reduce the max gas refunded after a transaction to `gas_used // MAX_REFUND_QUOTIENT`
+
+Remark: Ethereum Yellow Paper introduced the _max gas refunded_ in [(65)](https://github.com/ethereum/yellowpaper/blob/constantinople/Paper.tex#L648) as `gas_used // 2`. Here we
```

Comment:
Don't link to external GitHub repositories.  We *may* be able to make an exception for the yellow-paper, but it would be preferable if we just avoided it all together.

In this case, is this remark necessary?  It feels like it isn't needed for the spec?

Reply:
So where should I put it?

Reply:
My suggestion would just be to leave the remark out entirely.  I'm not sure that it adds value.

Another option would be to retain the remark, but remove the reference to the yellow paper like:
```
Remark: Max gas refund was previously defined as `gas_used // 2`.  Here we...
```

A third option would be to include an excerpt from the yellow paper inline.  Part of the propblem is that the yellowpaper is mutable, and it may not always live in that particular location in that particular GitHub repository.